src/strawberry_sqlalchemy_mapper/relay.py

```diff
@@ -146,6 +146,20 @@ async def resolve_async(nodes=nodes):
         return resolve_nodes(session)
 
 
+def exclude_relay(
```

Comment:
I wonder if this would be better defined in `mapper.type` as an argument?

Also, the name itself IMO implies you are excluding relay functionality, which is not totally true. My suggestion would be do the argument and call it `map_relations_as_list: bool = True`, in which you can opt-out as `False` (would be even better to be the other way around, but then it would be a breaking change)

Reply:
I could see a situation in which a type might have both relay and list. Would it make sense to add a class property in the same vein as `__exclude__`?